cmd/mt-fakemetrics/README.md

```diff
@@ -42,14 +42,56 @@ The speed is 100x what it would be if it were realtime (so a rate of 4x100x100=4
 mt-fakemetrics backfill --kafka-mdm-addr localhost:9092 --offset 5h --period 10s --speedup 100 --orgs 4 --mpo 100
 ```
 
-# Outputs
+## Value Policies
+
+You can specify a Value Policy with the `--value-policy` flag. Currently the following Value Policies are supported:
+
+* single
+* multiple
+* timestamp
+
+This flag can be useful when testing your deployment to ensure you have everything setup properly (i.e. aggregations, storage, etc...)
```

Comment:
I think i know what you mean with this sentence because i know that you're doing that to verify that cortextank returns the correct data. But without knowledge of that context i think this sentence is not very helpful.
I'd rather say something like "This flag can be useful when it is important to know what data fakemetrics generates to verify correctness of the data returned".